app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class LikesController < ApplicationController
+  def index
+    Like.create(author_id: params['author_id'], post_id: params['post_id'])
+    redirect_to '/'
+  end
```

Comment:
- [ ] you're supposed to use strong parameters for the params, have a look at [this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) for additional information.